sources/postgres/data.go

```diff
@@ -158,11 +158,14 @@ func convBytes(val string) ([]byte, error) {
 	return b, err
 }
 
-func convDate(val string) (civil.Date, error) {
+func convDate(conv *internal.Conv, val string) (interface{}, error) {
 	d, err := civil.ParseDate(val)
 	if err != nil {
 		return d, fmt.Errorf("can't convert to date: %w", err)
 	}
+	if conv.TargetDb == "experimental_postgres" {
```

Comment:
Suggest a comment along the lines of:

// In experimental_postgres mode, the civil.ParseDate call is only used as a validation step, and we
// throw its result away and return 'val'.

Reply:
Thanks for flagging this, this is non-trivial code indeed and deserved it's own explanation.